prefect_dask/task_runners.py

```diff
@@ -271,22 +283,25 @@ async def _start(self, exit_stack: AsyncExitStack):
             self.logger.info(
                 f"Connecting to an existing Dask cluster at {self.address}"
             )
-            connect_to = self.address
+            self._connect_to = self.address
         else:
             self.cluster_class = self.cluster_class or distributed.LocalCluster
 
             self.logger.info(
                 f"Creating a new Dask cluster with "
                 f"`{to_qualified_name(self.cluster_class)}`"
             )
-            connect_to = self._cluster = await exit_stack.enter_async_context(
+            self._cluster = await exit_stack.enter_async_context(
                 self.cluster_class(asynchronous=True, **self.cluster_kwargs)
             )
+            self._connect_to = self._cluster.scheduler_address
```

Comment:
I believe this will change behavior. Previously, it'd connect to `cluster` directly which is important for automatically Dask internally matching settings attached to the cluster object. Can you refactor this to retain the old call to `distributed.Client(...)` below?